clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -216,13 +216,16 @@ static bool isAAPCS(const TargetInfo &TargetInfo) {
   return TargetInfo.getABI().startswith("aapcs");
 }
 
-Address CIRGenFunction::getAddrOfField(LValue base, const FieldDecl *field,
-                                       unsigned index) {
+Address CIRGenFunction::getAddrOfBitFieldStorage(LValue base, 
+                                                 const FieldDecl *field,
+                                                 unsigned index, 
+                                                 unsigned size) { 
```

Comment:
Why are you changing the function signature? Why wasn't it `getAddrOfBitFieldStorage` in the first place?

Reply:
The function was introduced by me and in previous version it was not intended  to work with bitfields only. After the last changes it will serve only them though. That's why I renamed it and added one more argument

Reply:
Sounds good.